src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -380,6 +380,63 @@ public void SendAsync_ExpectedDiagnosticSourceActivityLogging()
             }, UseSocketsHttpHandler.ToString()).Dispose();
         }
 
+        [Fact]
+        public void SendAsync_ExpectedDiagnosticSourceActivityLogging_InvalidBaggage()
```

Comment:
This test uses async operations.  It should be changed to:

```c#
public async Task SendAsync_ExpectedDiagnosticSourceActivityLogging_InvalidBaggage()
```

Reply:
Hmm... I see you are doing this in a RemoteInvoke.  So, maybe it doesn't matter if it blocks since it is being done in a separate process.